lms/djangoapps/grades/models.py

```diff
@@ -791,3 +791,15 @@ def __unicode__(self):
     @classmethod
     def get_override_history(cls, override_id):
         return cls.objects.filter(override_id=override_id)
+
+
+class ScoreOverrider(models.Model):
+    """
+    Records who overrode a score, for bulk score assignments.
```

Comment:
Could you expand this with a little architectural decision record-y language around why not add this as a column on the corresponding table? e.g. just "we took this approach b/c adding columns to cwsm would be cray"

Reply:
How about [an ADR](https://github.com/edx/edx-platform/blob/8620b50396a29e093de2ba6730ce8bac9b4bc997/lms/djangoapps/grades/docs/decisions/0007-score-override-audit.rst)?

Reply:
A philosophical question: is the `grades` djangoapp/plugin the right place for this?  It doesn't relate to any of the other grade models, and most of the grading functions don't depend on it at all - only the csv processing stuff in grade_utils.py uses it.